docs/modules/getting_started/examples/code/hdfs.yaml

```diff
@@ -67,7 +44,7 @@ spec:
         selector:
           matchLabels:
             kubernetes.io/os: linux
```

Comment:
```suggestion
```

Reply:
There's another selector that I can't suggest an edit on that can also be removed (line 32)